packages/components/package.json

```diff
@@ -31,6 +31,7 @@
     "build:types": "tsc --project . --declaration --emitDeclarationOnly",
     "build:watch": "yarn run build:js --watch",
     "build:storybook": "build-storybook -o storybook-static",
+    "chromatic": "chromatic --project-token=CHROMATIC_PROJECT_TOKEN --skip 'dependabot/**' --only-changed",
```

Comment:
the `--only-changed` enables the [Turbosnap](https://www.chromatic.com/docs/turbosnap) feature where Chromatic "identifies component files that change, then intelligently builds and snapshots only the stories associated with those components ... However, we don’t recommend using TurboSnap immediately when starting out with Chromatic since the configuration is more complicated and can lead to difficult to debug scenarios or UI changes being missed. Instead, become familiar with Chromatic’s out of the box behavior and once your project has been running smoothly for some time consider trying out TurboSnap."
Should we use TurboSnap from the get go? It was working when I tried it out

Reply:
Let's establish a baseline first. We're still on the free plan so no worries about going over the limit right now. This also helps us know the upper bound of our usage.

Reply:
Got it, thanks for the advice, removed TurboSnap for now